### PR TITLE
doc: add initial section on how OpenJS helps projects

### PR DESCRIPTION
**Pull request by mhdawson**
Refs: https://github.com/nodejs/next-10/issues/110

Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

**Comment by mhdawson** [comment]
From https://github.com/nodejs/next-10/issues/110 would be good to start by inviting:

* JS Weekly
* Node Weekly
* ES.Next News
* Bytes

FYI @joesepi  and @rginn who I believe Joe talked to about this initiative.

**Comment by mhdawson** [comment]
@jorydotcom, @rginn hope we can discuss/agree on this in the next CPC meeting. I think @joesepi mentioned that he'd spoken to you and were up for something like this.

**Comment by rginn** [comment]
Kindly requesting we bump to May because our staff team is over capacity in April.

**Comment by mhdawson** [comment]
@rginn I don't think we need to block landing/agreeing on this, the start time can be independent.

**Review by mcollina — APPROVED** [review]
nice!

**Review by joesepi — APPROVED** [review]

File: support-for-projects/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Support for Projects
```
Comment (Abesyuffcjk) [inline support-for-projects/README.md:1]: A

Comment (Abesyuffcjk) [inline support-for-projects/README.md:1]: A

**Review by dylans — APPROVED** [review]

**Review by ljharb — APPROVED** [review]

**Comment by mhdawson** [comment]
Agreed we could land in the CPC meeting today.

**Comment by mhdawson** [comment]
@rginn I  believe we've onboarded the new communications person, would it be time to implement this.

**Comment by mhdawson** [comment]
@rginn just following up to see if this is something we can kick off now?

**Pull request merged**